### PR TITLE
CEDS-2194 - Skip 5/14 Country of Dispatch for C21

### PR DESCRIPTION
**Pull request by iservport**
For Clearance Requests, data element '5/14 Country of Dispatch' is not required in any circumstances.

### CODE REVIEW

**Comment by SteveSugden** [comment]
Is that it?  No tests? (Either fixed or new).
I was expecting to see the affected controllers being modified to indicate which journey types are valid, but looking at the code, we don't seem to do that in many places, which is a shame.   See for example https://github.com/hmrc/customs-declare-exports-frontend/blob/fa6854647e3634c3e0d06e23d60651d530fadc4c/app/controllers/declaration/CarrierDetailsController.scala#L47

At the very least you'll have to fix the acceptance tests I guess.

**Comment by SteveSugden** [comment]
Also (before anyone else comments) you need to fix the back-button behaviour which will mean a change in the Navigator class.

**Comment by bambuchaAdm** [comment]
Exactly as @SteveSugden said:
+ missing navigation fixes - on back on next page you would back to page you try to skip
and this means please fix tests for this controller they don't cover different type of journey

File: app/controllers/navigation/Navigator.scala [hunk 1]
```diff
@@ -102,13 +102,12 @@ object Navigator {
...
     case ContainerAdd                => controllers.declaration.routes.TransportContainerController.displayContainerSummary
-    case Document                    => controllers.declaration.routes.OfficeOfExitController.displayPage
-    case OriginationCountryPage      => controllers.declaration.routes.DeclarationHolderController.displayPage
+    case Document                    => controllers.declaration.routes.NatureOfTransactionController.displayPage
     case DestinationCountryPage      => controllers.declaration.routes.OriginationCountryController.displayPage
     case RoutingQuestionPage         => controllers.declaration.routes.DestinationCountryController.displayPage
     case RemoveCountryPage           => controllers.declaration.routes.RoutingCountriesSummaryController.displayPage
     case ChangeCountryPage           => controllers.declaration.routes.RoutingCountriesSummaryController.displayPage
-    case GoodsLocation               => controllers.declaration.routes.DestinationCountryController.displayPage
+    case GoodsLocation               => controllers.declaration.routes.DeclarationHolderController.displayPage
```
Comment (PatrykRudnicki) [inline app/controllers/navigation/Navigator.scala]: Hmmm, why this change? I think that Goods Location should redirect to the `DestinationCountryController`.

The redirection from Destination Country should be to Declaration Holder
